ui/component/settingContent/view.jsx

```diff
@@ -184,17 +182,6 @@ export default function SettingContent(props: Props) {
                     navigate={`/$/${PAGES.SETTINGS_BLOCKED_MUTED}`}
                   />
                 </SettingsRow>
-
-                {myChannelUrls && myChannelUrls.length > 0 && (
-                  <SettingsRow title={__('Creator settings')}>
-                    <Button
-                      button="inverse"
-                      label={__('Manage')}
-                      icon={ICONS.ARROW_RIGHT}
-                      navigate={`/$/${PAGES.SETTINGS_CREATOR}`}
-                    />
-                  </SettingsRow>
-                )}
```

Comment:
maybe could keep this but use the redirect link to the channel page instead? since people will still go to settings looking for `Creator Settings` either naturally ("Where do I find my settings?") or if they were used to it before, also `Settings` should group everything the user can modify so even if something is in a different place, they still know it's an option just by looking at this "general purpose" page

Reply:
> also Settings should group everything the user can modify

Should...yes, but they never did :smile: One of the main reasons why I wanted to move it is because the `Settings` page never contained all the channel related settings. From a technical perspective you could argue that these settings were separate because we use 2 different methods to store values, but our users don't really care about these details. Many creators didn't even know about the creator settings because all the other channel related settings were on the channel page itself.

Removing them entirely from the `Settings` might be bad in the short run, but I don't want creators to get used to another shortcut / link. The general idea is to move everything to a creator dashboard over the next few months, including better analytics and memberships. That's why the user settings and channel settings are now completely separated.